src/components/ModalExport.tsx

```diff
@@ -61,7 +62,8 @@ const ModalExport = () => {
     filterFunction: !shouldIncludeMetaAttributes ? child => !isFunction(child.value) : undefined
   }).length
 
-  const exportWord = 'share' in navigator ? 'Share' : 'Download'
+  const { isMobile } = DetectUserDevice()
+  const exportWord = isMobile ? 'Share' : 'Download'
```

Comment:
I would prefer a simpler touch detection mechanism. You can use [isTouch](https://github.com/cybersemics/em/blob/dev/src/browser.ts#L6) which is already defined.

Reply:
I saw existing `isTouch`. But won't it also be true for touch screen laptops? 🤔

Reply:
Yes, but I don't want to risk a false negative on mobile. It's a reasonable approach though.